raiden/network/rpc/client.py

```diff
@@ -129,10 +129,19 @@ def geth_discover_next_available_nonce(web3: Web3, address: AddressHex) -> Nonce
 
 
 def check_address_has_code(
-    client: "JSONRPCClient", address: Address, contract_name: str = "", expected_code: bytes = None
+    client: "JSONRPCClient",
+    address: Address,
+    contract_name: str = "",
+    expected_code: bytes = None,
+    given_block_identifier: BlockSpecification = "latest",
```

Comment:
I'd like to remove the default here, so we will not shoot ourselves in the foot.